doc/api/http.md

```diff
@@ -764,6 +764,27 @@ req.once('response', (res) => {
 });
 ```
 
+### request.writableFinished
+<!-- YAML
+added: v12.7.0
+-->
+
+* {boolean}
+
+Is `true` if all data has been flushed to the underlying system, immediately
+before the [`'finish'`][] event is emitted.
+
+### request.writableEnded
```

Comment:
This section needs to be placed before `### request.writableFinished` alphabetically.